app/src/Blocks/Column/Item.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace SilverStripe\Bambusa\Blocks\Column;
+
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
+use SilverStripe\Forms\RequiredFields;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Versioned\Versioned;
+
+class Item extends DataObject
+{
+    /**
+     * @var string
+     */
+    private static $singular_name = 'Item';
+
+    /**
+     * @var string
+     */
+    private static $table_name = 'ColumnItem';
+
+    /**
+     * @var array
+     */
+    private static $extensions = [
+        Versioned::class
+    ];
+
+    /**
+     * @var array
+     */
+    private static $db = [
+        'Sort' => 'Int',
+        'Content' => 'HTMLText',
+    ];
+
+    /**
+     * @var array
+     */
+    private static $has_one = [
```

Comment:
Please ensure this works well with fluent. Some pointers:
https://github.com/silverstripe/bambusa-installer/pull/44
https://github.com/silverstripe/bambusa-installer/blob/master/app/_config/fluent.yml

Reply:
right, I think I missed that completely...

Reply:
Yeah no worries, that's very easy to miss. Good learning excercise for how our modules interact!

Reply:
Fixed. Tested the fluent stuff on top of #44, but the stuff in this PR is standalone enough so can be considered separately.